### PR TITLE
Setting Kubecon 2022 NA banner to show on 2022-09-23

### PR DESCRIPTION
**Pull request by nate-double-u**
Follow up to https://github.com/kubernetes/website/pull/36107

The CNCF has asked if we can show the KubeCon 2022 NA banner sooner than was initially set. Showing the banner for a few weeks ahead of KubeCon would help a lot with awareness and attendance.

/committee steering
/cc @reylejano @natalisucks @sftim @mrbobbytables @dims @liggitt @cblecker

### CODE REVIEW

File: data/announcements/scheduled.yaml [hunk 1]
```diff
@@ -118,7 +118,7 @@ announcements:
     May 16 - 20, 2022.
 
 - name: Kubecon 2022 NA
-  startTime: 2022-10-17T00:00:00 # Added in https://github.com/kubernetes/website/pull/36107 
+  startTime: 2022-09-23T00:00:00 # Added in https://github.com/kubernetes/website/pull/36107
```
Comment (sftim) [inline data/announcements/scheduled.yaml:121]:
```suggestion
  startTime: 2022-09-23T00:00:00 # Added in https://github.com/kubernetes/website/pull/36107
                                 # Date amended in https://github.com/kubernetes/website/pull/36927
```
?

**Review by sftim — COMMENTED** [review]
/lgtm

This is a valid ISO 8601 date.

File: data/announcements/scheduled.yaml (hunk 1 above)
Comment (nate-double-u) [reply data/announcements/scheduled.yaml]: oops, thx

Comment (nate-double-u) [reply data/announcements/scheduled.yaml]: There are a few of these in this file -- we may want to go and do a cleanup pass at some point.

**Comment by reylejano** [comment]
/approve
@kubernetes/steering-committee please unhold when adequate

**Comment by natalisucks** [comment]
/lgtm
/remove-hold Given that Steering Committee reps have given LGTMs. Thanks all and thanks @nate-double-u 🙌

**Pull request merged**